docs/src/components/TitleSection.tsx

```diff
@@ -0,0 +1,29 @@
+/*!
```

Comment:
This file is named using `PascalCase`, but should be `camelCase`. Case sensitivity handling will make it difficult to just push a name change here. But once we're more-or-less ready to merge, we could rename this file to something temporary like `foo.tsx`, push, and then rename back to `titleSection.tsx`.

Reply:
But we should avoid landing this in the main branch without fixing the name first.